apps/example-server/src/main/case/case.routes.ts

```diff
@@ -12,8 +12,46 @@ const secret = process.env.ACCESS_TOKEN_SECRET ?? ''
 
 const authMiddleware = jwt({ secret })
 
-// GET /cases - list all
-app.get('/cases', authMiddleware, async (c) => {
+const checkCaseOwnershipMiddleware = async (ctx: Context, next: Next) => {
+    const payload = await ctx.get('jwtPayload')
+    if (!payload) {
+        return ctx.json(
+            { error: 'Unauthorized', message: 'Not authenticated' },
+            403,
+        )
+    }
```

Comment:
We will check this part by the `authMiddleware` or (`jwt({ secret })`) middleware. So this part can be removed.